signatory-ledger-tm/Cargo.toml

```diff
@@ -1,11 +1,11 @@
 [package]
-name        = "signatory-ledger-cosval"
-description = "Signatory provider for ledger cosmos validator app"
-version     = "0.11.0-pre" # Also update html_root_url in lib.rs when bumping this
+name        = "signatory-ledger-tm"
+description = "Signatory provider for Ledger Tendermint Validator app"
+version     = "0.11.0" # Also update html_root_url in lib.rs when bumping this
```

Comment:
looking into lib.rs, it still says 
`html_root_url = "https://docs.rs/signatory-ledger-cosval/0.11.0"` 
Not sure if this also needs renaming.